awesome.c

```diff
@@ -801,13 +801,15 @@ main(int argc, char **argv)
     globalconf.have_composite = query && query->present;
     if (globalconf.have_composite)
         xcb_discard_reply(globalconf.connection,
-                xcb_composite_query_version(globalconf.connection, 0, 4).sequence);
+                xcb_composite_query_version(globalconf.connection, 0, 3).sequence);
```

Comment:
This change ended up in the wrong commit, but whatever...

Reply:
oops, I forgot there were two commits :p